src/domain/alerts/mappers/multi-send.mapper.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@nestjs/common';
+import {
+  decodeFunctionData,
+  getAddress,
+  Hex,
+  hexToBigInt,
+  hexToNumber,
+  parseAbi,
+  size,
+  slice,
+} from 'viem';
+
+@Injectable()
+export class MultiSendMapper {
```

Comment:
No strong opinion, but we already have _mappers_ in the project with a slightly different purpose than this class: to aggregate/transform domain entities into route/presentation entities. What about using another suffix for this one? I think _decoder_ could be an option. I left this up to you. If you prefer to maintain the _mapper_ suffix, I think it's also fine.

Reply:
I've renamed this to `MultiSendDecoder` and moving the files/adjusted the tests accordingly in f78093192d8c7d5555f13003c7548c368e3a20b0.